examples/extend_via_sc_schema/extend_via_schema.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from omegaconf import OmegaConf
+
+import hydra
+from hydra.core.config_store import ConfigStore
+from hydra_configs.torch.optim import AdamConf
+
+
+@dataclass
+class Config:
+    optim: AdamConf
+    debug: bool = False
+
+
+cs = ConfigStore.instance()
+cs.store(name="base_config", node=Config)
+cs.store(group="optim", name="base_adam", node=AdamConf)
```

Comment:
We don't have a way for the user to get the config registered? (something like hydra_configs.torch.register_configs()?`
Why are you registering it here?

Reply:
Are you implying we consolidate lines 10-18 into a single call:
`hydra_configs.torch.register_configs()`?

Reply:
Not exactly.
Registering a schema for your own config is one thing (line 17), but registering AdamConf (and the rest of the configs hydra-torch is providing) is something we can provide a function (or functions as we have discussed) to do.

This app:
- defines its own Config class.
- registers it
- registers a config class defined by the library.
I am suggesting that registering config classes defined by the library will be done by the library and will be triggered with a function call.

This will enforce a consistent naming convention for the registered classes and will reduce the boilerplate required to use the configs package.

Reply:
Great, I agree. I'll make a quick PR for the registration. Then we can use that in this example.